nwnsc/nwnsc.cpp

```diff
@@ -333,7 +333,7 @@ Return Value:
 #if defined(__APPLE__)
                 SteamRootDir = GetHomeDirectory() + "/Library/Application Support/Steam/steamapps/common/Neverwinter Nights/";
 #elif defined(__linux__)
-                SteamRootDir = GetHomeDirectory() + "/Library/Application Support/Steam/steamapps/common/Neverwinter Nights/";
+                SteamRootDir = GetHomeDirectory() + "/.local/share/Steam/steamapps/common/Neverwinter Nights/";
 #elif defined(_WINDOWS)
                 SteamRootDir = GetHomeDirectory() + "/Library/Application Support/Steam/steamapps/common/Neverwinter Nights/";
```

Comment:
Can you also add `"C:\\Program Files (x86)\\Steam\\steamapps\\common\\Neverwinter Nights\\"` (no home directory) for Windows here?